### PR TITLE
LPS-172428 [PRM] Add better Responsiveness to Dashboard Page

### PR DESCRIPTION
**Pull request by JoMaAlves**

### CODE REVIEW

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/dashboard-collection/common/components/donut-chart/index.js [hunk 1]
```diff
@@ -35,77 +35,100 @@ const DonutChart = ({
 		}));
 	}, []);
 
+	const hasChartData = chartData.columns.filter((column) => column[1]).length;
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/dashboard-collection/common/components/donut-chart/index.js:38]:
```suggestion
	const hasChartData = useMemo(() => chartData.columns.filter((column) => column[1]).length, [...]);
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:forward

**Pull request closed without merging**